includes/class-woocommerce-custom-orders-table-cli.php

```diff
@@ -345,4 +319,40 @@ protected function get_order( $order_id ) {
 
 		return $order;
 	}
+
+	/**
+	 * Build a SQL query to get posts that require migration.
+	 *
+	 * @global $wpdb
+	 *
+	 * @param string $select The contents of the SELECT clause.
+	 * @param int    $limit  The maximum number of results to return, or '0' to not limit the number
+	 *                       of results.
+	 * @param int    $offset Optional. The offset value. Default is 0.
+	 *
+	 * @return string The prepared SQL query.
+	 */
+	protected function get_migration_query( $select, $limit, $offset = 0 ) {
```

Comment:
Does it make sense to set a default select clause or limit here?

Reply:
I could go either way, but I kind of prefer making the calls explicit.

Once we get past the 1.0.0 release (especially as we start moving towards using the Action Scheduler), I want to refactor all of the migration utilities into a separate class.